api/v1alpha3/virtualmachine_storage_types.go

```diff
@@ -90,3 +126,40 @@ type VirtualMachineVolumeStatus struct {
 	// volume.  Error will be empty if attachment succeeds.
 	Error string `json:"error,omitempty"`
 }
+
+// VirtualMachineStorageStatus defines the observed state of a VirtualMachine's
+// storage.
+type VirtualMachineStorageStatus struct {
+
+	// +optional
+
+	// ChangeBlockTracking describes whether or not change block tracking is
+	// enabled for the VirtualMachine.
+	// ChangeBlockTracking *bool `json:"changeBlockTracking,omitempty"`
```

Comment:
Still TBD if we're moving this here, or just hadn't gotten around to writing the conversion code yet?

Reply:
I want to move it, but I'm hesitant to do it in *this* PR. Originally I had moved it, along with CBT, to the new status.storage. But then I realized that only really makes sense if we move spec.cbt and spec.volumes to spec.storage as well, and I did not think we should combine those changes in this PR.